lib/phone_number.js

```diff
@@ -0,0 +1,39 @@
+var phone = require('./locales/common/phone');
+
+/**
+ * namespace ngFaker.phone
+ */
+function Phone(ngFaker) {
+  /**
+   * returns a random phone number
+   *
+   * @method phoneNumber
+   * @returns {string} random phone number
+   */
+  this.phoneNumber = function (options) {
+    var phoneNumber;
+
+    if (options && options.addInternationalPrefix) {
```

Comment:
I think `addInternationalPrefix` is too long, we can use a number to toggle instead, like if the user enters 0 it returns international format.

Or we use the options object but use `intl` as the key instead so `options = { intl: true }`, something like that.